test_wcd2reshare.py

```diff
@@ -110,6 +121,23 @@ def test_fields_of_no_interest_ignored():
     assert wcd2reshare.query_formatter(params) == "type=ISN&lookfor=978-3-16-148410-0"
 
 
+def test_webhook_configures_sentry_if_dsn_present(caplog, monkeypatch):
+    monkeypatch.setenv("SENTRY_DSN", "https://1234567890@00000.ingest.sentry.io/123456")
+    caplog.set_level(logging.INFO)
```

Comment:
Rather than setting the caplog level within individual tests, you can add it to setup.cfg like this:
```
[tool:pytest]
log_level = INFO
```

I would also remove the empty `[mypy]` section in setup.cfg unless it's needed for some reason?